src/components/TransactionSearch/style.css

```diff
@@ -1,23 +1,18 @@
 @import '../../vars.css';
 
 .mzw-transaction-search__container {
-  min-width: 15rem;
+  min-width: 15em;
 }
 
 .mzw-transaction-search__label {
-  margin-bottom: 0.4rem;
+  margin-bottom: 0.4em;
   color: var(--color-light-grey);
 }
 
 .mzw-transaction-search__input {
-  padding: 0.5rem 1rem;
+  padding: 0.5em 1em;
```

Comment:
Why are these `em` now?

Reply:
rem is relative to the root element so doesn't scale based on the other elements